libcodechecker/cmd/cmd_line_client.py

```diff
@@ -163,6 +163,8 @@ def setupClient(host, port, uri):
                     print("Authenticated using pre-configured credentials...")
                 except shared.ttypes.RequestFailed:
                     print_err = True
+            else:
```

Comment:
Please move the changes non relevant to the restructuring into a separate commit